extension/serializer/src/test/kotlin/metadata/AvroMetaDataExtTest.kt

```diff
@@ -0,0 +1,37 @@
+package io.holixon.axon.avro.serializer.metadata
+
+import io.holixon.avro.adapter.api.AvroAdapterApi.schemaResolver
+import org.assertj.core.api.Assertions.assertThat
+import org.axonframework.messaging.MetaData
+import org.junit.jupiter.api.Test
+
+internal class AvroMetaDataExtTest {
+
+  @Test
+  internal fun `avro meta to axon meta`() {
+    val avro = AvroMetaData.newBuilder()
+      .setValues(
+        mapOf(
+          "foo" to "bar"
```

Comment:
Please test more value types... is it only `String` and `int`? Do we have a problem with `int` `Int`and `Integer`?
What does AxonIQ do in GRPC?